rel/config.exs

```diff
@@ -6,40 +6,3 @@
 |> Path.join()
 |> Path.wildcard()
 |> Enum.map(&Code.eval_file(&1))
-
-use Distillery.Releases.Config,
-  # This sets the default release built by `mix distillery.release`
-  default_release: :default,
-  # This sets the default environment used by `mix distillery.release`
-  default_environment: :prod
-
-# For a full list of config options for both releases
-# and environments, visit https://hexdocs.pm/distillery/config/distillery.html
-
-# You may define one or more environments in this file,
-# an environment's settings will override those of a release
-# when building in that environment, this combination of release
-# and environment configuration is called a profile
-
-environment :prod do
-  set(include_erts: true)
```

Comment:
@paulswartz I tried moving all these `set()` lines to the `releases` file at first, but it didn't seem happy about that. Screens doesn't seem to have any of these. Is it ok for these to go away, or is there something we need to carry forward in a new way?

Reply:
`include_erts: true` is the default, `include_src` isn't a feature of the native releases, and the cookie is fetched from `RUNTIME_COOKIE` by default. So the only thing you need to do here is switch the environment variable with the cooking from ERLANG_COOKIE to RUNTIME_COOKIE.

Full documentation: https://hexdocs.pm/mix/1.9.0/Mix.Tasks.Release.html#module-runtime-configuration

Reply:
There's also some stuff in `rel/vm.args` that you can either port over or delete the file.